src/pytom_tm/tmjob.py

```diff
@@ -332,6 +336,13 @@ def __init__(
         self.start_slice = 0
         self.steps_slice = 1
         self.rotational_symmetry = rotational_symmetry
+        self.particle_diameter = particle_diameter
+        # calculate increment from particle diameter
+        if angle_increment is None and particle_diameter is not None:
```

Comment:
should we update the default angle increment to `None` instead of `7.0` and set it to `7.0` during the init?